lib/core/src/Cardano/Wallet.hs

```diff
@@ -468,6 +470,31 @@ readWallet ctx wid = db & \DBLayer{..} -> mapExceptT atomically $ do
   where
     db = ctx ^. dbLayer @s @k
 
+
+-- Rather than force all callers of 'readWallet' to wait for fetching the
+-- account balance (via the 'NetworkLayer'), we expose this function for it.
+fetchWalletRewardBalance
```

Comment:
**[suggestion]**

Drop `Wallet` from the name

**Why**

There's no much ambiguity about where the reward balance comes from. It makes the overall clearer.